src/main/java/com/ericsson/ei/subscription/InformSubscriber.java

```diff
@@ -89,13 +89,26 @@
     private JmesPathInterface jmespath;
 
     @Autowired
-    private SendHttpRequest restTemplate;
+    private HttpRequestSender restTemplate;
```

Comment:
Please rename the variable name as well. This was actually renamed sendHttpRequest once but must have gotten it´s old old name in a merge.

Reply:
You mean the restTemplate? :) will do that